pyanaconda/modules/storage/partitioning/interactive/utils.py

```diff
@@ -668,19 +668,19 @@ def collect_file_system_types(device):
     """
     # Collect the supported filesystem types.
     supported_types = {
-        fs.name for fs in get_supported_filesystems()
-        if fs.name not in UNSUPPORTED_FILESYSTEMS
+        fs.type for fs in get_supported_filesystems()
+        if fs.type not in UNSUPPORTED_FILESYSTEMS
```

Comment:
You ale switching from names to types but in ``:return:`` is talking about ``..system names``

Reply:
Fixed.